Assets/PatchKit/UnityEditorExtension/Editor/Core/AppBuild.cs

```diff
@@ -97,18 +92,19 @@ public static string Location
         }
     }
 
-    private static bool AreThereOnlyBuildEntries(
+    [NotNull]
+    private static IEnumerable<string> FilesOutsideOfBuildEntries(

```

Comment:
Since this is a function, this should be called "GetFilesOutsideOfBuildEntries"

Reply:
Done